### ISSUE TITLE
??

### ISSUE THREAD

**Issue by kyramari** [kyramari/icon-group#1]
<img width="502" alt="screen shot 2016-03-01 at 1 43 40 pm" src="https://cloud.githubusercontent.com/assets/16653506/13527720/2aecdbe6-e1df-11e5-9fdb-b22857925299.png">

So this is how far I got. I can't seem to figure out how to get the background around the image as well. I have tried a bunch of different ways and none work. 

@thomasjbradley

**Comment by thomasjbradley** [comment]
The `.icon-group-item` class should be around both the image and the text, because if you look at the CSS, that's the thing that has the grey background on it.

**Comment by kyramari** [comment]
<img width="417" alt="screen shot 2016-03-04 at 11 08 07 am" src="https://cloud.githubusercontent.com/assets/16653506/13532506/d9fed3a4-e1f9-11e5-819e-e35455912d81.png">

So I put .icon-group-item into the <div> but it makes the image go on a separate line for some reason.

**Comment by thomasjbradley** [comment]
Yeah... remember in the sections code we had a `<div>` around the image and around the text.
The same should be done here, then each of those `<div>` tags need a class on them.

**Comment by kyramari** [comment]
ohhhhh! thanks

**Comment by kyramari** [comment]
<img width="439" alt="screen shot 2016-03-04 at 12 20 16 pm" src="https://cloud.githubusercontent.com/assets/16653506/13534507/80c1daf2-e203-11e5-82c5-7c19b8be89eb.png">

So now I have this. But I can't get the text to not align centre.

**Comment by thomasjbradley** [comment]
It's the `.icon-group` class, here:
https://github.com/kyramari/icon-group/blob/gh-pages/index.html#L37

It should be around all of them.

**Comment by kyramari** [comment]
So I did this now. But when the screen gets larger they do not realign and go to the top. They just stay in a column.
Also, this is what Markbot is saying:
<img width="493" alt="screen shot 2016-03-04 at 2 18 51 pm" src="https://cloud.githubusercontent.com/assets/16653506/13538676/f52c747c-e219-11e5-8fe3-a0aa7c1df516.png">

**Comment by thomasjbradley** [comment]
There are two problems:
1. The `<ul>` should surround all of them—one single `<ul>` for all.
2. The `<div>` with `.icon-group-item` should be `<li>` tags instead, like here:
   https://github.com/kyramari/icon-group/blob/gh-pages/index.html#L12

**Comment by kyramari** [comment]
So I did that. But it looks like this:
<img width="505" alt="screen shot 2016-03-04 at 5 42 59 pm" src="https://cloud.githubusercontent.com/assets/16653506/13542412/9a44d402-e230-11e5-9e37-f833932f7633.png">

Markbot:
![screen shot 2016-03-04 at 5 39 59 pm](https://cloud.githubusercontent.com/assets/16653506/13542420/ab5e2658-e230-11e5-8974-757e263c268d.png)

**Comment by thomasjbradley** [comment]
These can't be `<li>` tags because they aren't inside a `<ul>` or `<ol>`—they’re inside a `<div>`.
https://github.com/kyramari/icon-group/blob/gh-pages/index.html#L17-L18

**Comment by kyramari** [comment]
ok
